src/costmap/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>costmap</name>
+  <version>0.1.0</version>
+  <description>TODO</description>
+  <author email="oregon.robotics.team@intel.com">Oregon Robotics Team</author>
+   <author email="carlos.a.orduno@intel.com">Carlos Orduno</author>
```

Comment:
Indentation is off. Also should discuss (again) how we're handing author and maintainer fields.

Reply:
Added to list of discussion topics.